Doc/library/turtle.rst

```diff
@@ -14,6 +14,11 @@
    from turtle import *
    turtle = Turtle()
 
+.. testcleanup::
+
+   import os
+   os.remove("my_drawing.ps")
```

Comment:
Two birds with one stone; the `save` feature is new in 3.14, and this works to kill the window after the tests for me locally:

```suggestion
   # Destroy the turtle window after tests are complete
   # Imported via star import in testsetup
   bye()
```

Reply:
https://github.com/python/cpython/pull/125294#issuecomment-2407024482 -- this doesn't seem to do anything for me locally :/

for now I'll just revert the changes to `turtle.rst` and only backport the `configparser.rst` changes